KsApi/models/templates/ProjectCreatorDetailsEnvelopeTemplate.swift

```diff
@@ -0,0 +1,9 @@
+import Prelude
+
+extension ProjectCreatorDetailsEnvelope {
+  internal static let template = ProjectCreatorDetailsEnvelope(
+    backingsCount: 10,
+    id: "",
```

Comment:
Thanks for adding the template, we could probably give it some arbitrary value for `id` like `"123"`.